sdk/monitor/azure-monitor-opentelemetry/dev_requirements.txt

```diff
@@ -3,6 +3,6 @@ pytest
 django
 fastapi-slim
 flask
-psycopg2
+psycopg2-binary
```

Comment:
Would you mind adding a comment here linking to the issue so we have a history of why we used this?